app/helpers/courses_helper.rb

```diff
@@ -8,9 +8,19 @@ def load_course_management course
   end
 
   def load_user_is_traine course_id
-    user_ids = CourseManagement.load_course_managements(course_id)
-                               .pluck(:user_id)
-    @user_ids_not = User.load_user_not(user_ids).traine
-                        .pluck(:full_name, :id)
+    user_ids = CourseManagement
+               .load_course_managements_and_has(course_id)
+               .activited
+               .pluck(:user_id)
+    if user_ids.blank?
```

Comment:
ko dùng if else, dùng toán tử ngôi số 3 đi

Reply:
e dùng return if thay thế được không chị ơi . cái biến e gọi nhiều dòng khi gọi  ba ngôi nó quá dòng nên nó tự gợi ý thành dùng if else chị

Reply:
dc